WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -77,25 +85,38 @@ struct CartView: View {
             Spacer()
             switch viewModel.orderStage {
             case .building:
-                checkoutButton
-                    .padding(Constants.checkoutButtonPadding)
+                if cartViewModel.isCartEmpty {
+                    EmptyView()
+                }
+                else {
+                    checkoutButton
+                        .padding(Constants.checkoutButtonPadding)
+                }
             case .finalizing:
                 EmptyView()
             }
         }
         .frame(maxWidth: .infinity)
-        .background(Color.posBackgroundWhitei3)
+        .background(cartViewModel.isCartEmpty ? Color.posBackgroundEmptyWhitei3.ignoresSafeArea(edges: .all) : Color.posBackgroundWhitei3.ignoresSafeArea(.all))
     }
 }
 
 private extension CartView {
     enum Constants {
         static let primaryFont: Font = .system(size: 40, weight: .bold, design: .default)
-        static let secondaryFont: Font = .system(size: 20, weight: .semibold, design: .default)
-        static let checkoutButtonPadding: CGFloat = 32
+        static let secondaryFont: Font = .system(size: 24, weight: .regular, design: .default)
```

Comment:
Do we care about the dynamic type in the scope of POS? Since all these fonts won't change when accessibility font size increase

Reply:
Created https://github.com/woocommerce/woocommerce-ios/issues/13426